net.devolutions.RDM.yml

```diff
@@ -0,0 +1,38 @@
+app-id: net.devolutions.RDM
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: remotedesktopmanager
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission

Reply:
If there's a way to limit to the home directory, that would fit our need.

Reply:
It's in the documentation

https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html?highlight=filesystem#filesystem-permissions

But even just the home dir is too much IMHO